client/my-sites/plan-features/style.scss

```diff
@@ -230,13 +238,23 @@ $plan-features-header-banner-height: 20px;
 	margin: 0;
 	font-size: 14px;
 	color: $gray;
+	position: relative;
 
 	&.is-placeholder {
 		@include placeholder( 23% );
 		height: 50px;
 		margin-top: 5px;
 	}
 }
+.plan-features__footer-desc-longest {
+	visibility: hidden;
```

Comment:
I'm not sure what will Google search indexer think about this. How it impacts the SEO. On the other hand, this content is hidden behind auth anyways so Google won't index it. But what if the plans features component is used on an indexable place?